components/Layout.js

```diff
@@ -0,0 +1,15 @@
+import Navbar from "./Navbar";
+import Image from "next/image";
+import Footer from "./Footer";
+
+const Layout = ({ children }) => {
+  return (
+    <div className="bg-gradient-to-tr from-[#a8c1ec] to-[#E1EBFA]">
```

Comment:
Hello Mr @MugaboDenys , everything looks good according to the things that we should check while reviewing this project and you are good to go 🚀, but I could encourage you to avoid using Arbitrary Value as much as possible while working with TailwindCSS, if you get time try to fix it!